Include/internal/pycore_object.h

```diff
@@ -30,6 +30,55 @@ _PyType_HasFeature(PyTypeObject *type, unsigned long feature) {
 extern void _PyType_InitCache(PyInterpreterState *interp);
 
 
+/* Immortal Objects
+ *
+ * An "immortal" object is one for which Py_DECREF() will never try
+ * to deallocate it.
+ *
+ * At the moment this API is strictly internal.  However, if it proves
+ * helpful for extension authors we may move it to the public API. */
+
+#define _Py_IMMORTAL_OBJECTS 1
```

Comment:
I don't get the purpose of this macro since it's always defined.

Reply:
It's there only as a marker relative to the public API.  However, it is not necessary at this point and I'll drop it.